lib/topic_view.rb

```diff
@@ -470,8 +470,16 @@ def participant_count
 
   def participants
     @participants ||= begin
+      participant_users = User.
+        where(id: post_counts_by_user.keys).
+        includes(:primary_group)
```

Comment:
I _think_ our styleguide favors leading `.` over trailing dots.

```suggestion
      participant_users = User
        .where(id: post_counts_by_user.keys)
        .includes(:primary_group)
```